csharp_dotnetcore/4.Simple-Prompt-Bot/BotAccessors.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace Simple_Prompt_Bot
+{
+    public class BotAccessors
```

Comment:
Might steal what we did in sample 2, we created a constructor that took the State object so we could signal that the State object should be accessible from the bot to enable explicit Save operations (we removed the BotStateSet).